File: platform/sdk/nodejs/examples/graph/graph-005_cdc_dedup_stats.js
const { GraphClient } = require("../../xuanji-sdk-graph");
const client = new GraphClient({ endpoint: "graph.local" });
client.cdcNew("consumer-005");
const result = client.cdcDedupStats("consumer-005");
if (!result.ok || result.totalEvents <= 0) process.exit(1);
console.log("XJ-OK: graph-005_cdc_dedup_stats");
process.exit(0);

File: platform/sdk/python/examples/graph/graph-005_cdc_dedup_stats.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from mox_sdk_graph import GraphClient

client = GraphClient({"endpoint": "graph.local"})
client.cdcNew("consumer-005")
result = client.cdcDedupStats("consumer-005")
if not result["ok"] or result["totalEvents"] <= 0:
    sys.exit(1)
print("XJ-OK: graph-005_cdc_dedup_stats")
sys.exit(0)
